### ISSUE TITLE
Retrigger Async Ad Loading

### ISSUE THREAD

**Issue by r3vile** [revive-adserver/revive-adserver#725]
It would be great to be able to retrigger async ad loading, e.g. in case an ad should be displayed after an ajax call or after a short period of time.

**Comment by imsus** [comment]
I've found a workaround for this.

**tl;dr**
Run this code after you change the view in SPA.
`reviveAsync.{your-data-revive-id}.apply(reviveAsync.{your-data-revive-id}.detect());`

The `asyncjs.php` files expose a global `reviveAsync` object. Which furthermore we can explore this variable.

![screen shot 2016-07-21 at 11 59 05](https://cloud.githubusercontent.com/assets/1058471/17011708/c50c2a4c-4f3a-11e6-87ef-d5980611c5f0.png)

I got this `apply` and `detect` function by pretty print the `asyncjs.php` response in chrome. Look for line 16.

![screen shot 2016-07-21 at 12 10 43](https://cloud.githubusercontent.com/assets/1058471/17011877/3a2bc1ce-4f3c-11e6-8051-b1678ed8964a.png)

**Comment by athlan** [comment]
@imsus thank you for investigation.

If the `data-revive-id begins` with number, you have to call the property of object in different way:
```
reviveAsync['8788d5bac004ca4b9a315d4954da6587'].apply(reviveAsync['8788d5bac004ca4b9a315d4954da6587'].detect());
```
